core/jobs.py

```diff
@@ -941,9 +945,9 @@ def validate(cls, unused_mapper_spec):
 
 
 class BaseMapReduceJobManagerForContinuousComputations(BaseMapReduceJobManager):
-    """Continuous Computation jobs which runs continuously, are used to perform
+    """Continuous Computation jobs, which runs continuously, are used to perform
```

Comment:
Continuous computation jobs, which run continuously in the background, are used ...

(note: decapitalize "Computation", runs --> run)

Reply:
done.